spec/requests/api/v2/app_groups_spec.rb

```diff
@@ -1,5 +1,79 @@
 require 'rails_helper'
 
+RSpec.describe 'Deactivated App Group by Cluster Name', type: :request do
+  let(:headers) do
+    { 'ACCEPT' => 'application/json', 'HTTP_ACCEPT' => 'application/json' }
+  end
+
+  before(:all) do
+    @access_token = 'E8cjTcUKucBxRs-XV_EvosUgMsytLGVBhmD_b6KpOVbYnhX3THgVtLMXOHS606Us'
```

Comment:
please don't put the actual access_token on the code. use an [example](https://github.com/BaritoLog/BaritoMarket/blob/efc95d40c83d68170756efc8638d50b9b20efd0b/spec/requests/api/v2/app_groups_spec.rb#L8-L11) instead.